src/protocol/protocol.ts

```diff
@@ -656,7 +682,8 @@ export namespace BrowsingContext {
     | LoadEvent
     | DomContentLoadedEvent
     | ContextCreatedEvent
-    | ContextDestroyedEvent;
+    | ContextDestroyedEvent
+    | ScriptBindingCalledEvent;
```

Comment:
It should be in `Script` namespace

Reply:
`MessageEvent` instead of Binding as per the other comment